phoenix_liveview/assets/app.css

```diff
@@ -0,0 +1,15 @@
+html, body {
```

Comment:
I have kept a bit of CSS to demonstrate the use of the `esbuild` pipeline. I could probably find a way to move it inside the "single file app", but it would generate some extra complexity which is probably not very relevant.